flight-search-be/src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=flight-search-be
+
+amadeus.api.client-id=Hdstq6c1Jxiyv6CDS5N7Amrt9P4unlUq
+amadeus.api.client-secret=AYEaG4BCcpWEIAxi
```

Comment:
Do not upload secrets to the repository. They could be environment variables that each developer has in their local environment